zesty/utils/rekordo/database.go

```diff
@@ -43,6 +46,10 @@ func RegisterDatabase(dbcfg *DatabaseConfig, tc gorp.TypeConverter) (zesty.DB, e
 		dbcfg.MaxIdleConns = maxIdleConns
 	}
 	dbConn.SetMaxIdleConns(dbcfg.MaxIdleConns)
+	if dbcfg.ConnMaxLifetime == time.Duration(0) {
```

Comment:
Prefer ```if dbcfg.ConnMaxLifetime <= 0``` to handle the negative value case.
(same as underlying implementation: https://golang.org/src/database/sql/sql.go?s=23062:23111#L812 )